src/Gir/Model/Include.cs

```diff
@@ -3,7 +3,6 @@
 
 namespace Gir
 {
-	[Serializable]
```

Comment:
Why remove the serializables here?

Reply:
It was only on 3 files and doesn't appear to be needed. I was just going for consistency